src/components/GoogleLoginButton.js

```diff
@@ -4,17 +4,15 @@ import styles from '../styles/Login.module.css';
 import { Button } from 'antd';
 import { GoogleOutlined } from '@ant-design/icons';
 import GoogleLogin from 'react-google-login';
-import axios from 'axios';
+import { connect } from 'react-redux';
+import { loginUser } from '../redux/actions/authActions';
+import '../App.css';
 
-const GoogleLoginButton = () => {
+const GoogleLoginButton = ({ auth: { user, isAuthenticated }, loginUser }) => {
   const { Title } = Typography;
```

Comment:
This line can be removed

Reply:
No we have to use this to destructure the props

Reply:
Okay.